tavla/app/(admin)/edit/[id]/compatibility.ts

```diff
@@ -0,0 +1,99 @@
+// TODO: remove 15. december when new lines are active
+// Remember to do a migration script in Firestore beforehand
+
+import { TBoard } from 'types/settings'
+import { TTile } from 'types/tile'
+
+export const SWITCH_DATE = Date.UTC(2024, 11, 15)
+
+export function makeBoardCompatible(board: TBoard): TBoard {
+    const updatedTiles = board.tiles.map(({ whitelistedLines, ...tile }) => ({
+        ...tile,
+        whitelistedLines: whitelistedLines?.flatMap(oldLineIdsToNew),
+    })) as TTile[]
+
+    return { ...board, tiles: updatedTiles }
+}
+
+function oldLineIdsToNew(lineId: string): string | string[] {
+    switch (lineId) {
+        case 'VYG:Line:41':
```

Comment:
veldig smak og behag, men kunne det gått å slenge alle disse i en dictionary med key gammel id og value ny, og legge den i en egen fil elns? så vil man kunne loope gjennom bare istedenfor å ha så mange cases. er kanskje ikke så viktig uansett siden all denne koden slettes etterhvert så feel free to ignore

Reply:
ja, tenkte det samme ☝️ . Generelt er vel lookup mer effektiv med map vs switch eller if-else som er store  🤔 

Siden devicer i dag er ganske bra, er det ikke så kritisk 💭 . 

Bare å sammenligne [her](https://jsben.ch/) f.eks.

Reply:
Jeg tok utgangspunkt i det Kent foreslo, men jeg ser litt mer på de ulike forslagene! 

Jeg får opp noe rare greier med linken @purusott :/

Reply:
var denne linken: https://jsben.ch/
det er bare en måte å sammenligne hvordan ulik kode kjører, så man kan paste inn og sammenligne hvor fort det kjører, men er ikke så viktig 😅

Reply:
ahhah den var kul!! resultat: 
![image](https://github.com/user-attachments/assets/28d401c8-7659-44d2-a1aa-08a458c677ae)

lar switchen ligge 😎